linux_os/guide/services/ssh/ssh_client/ssh_client_use_approved_ciphers_ordered_stig/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+{{%- if product == 'ubuntu2404' %}}
+{{%- set ssh_approved_ciphers="aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes128-ctr" %}}
```

Comment:
I think it would be better to define this constant at a single place, for example in `product.yml` instead of repeating it multiple times in various files.